pkg/virt-handler/ns/ns.go

```diff
@@ -0,0 +1,226 @@
+// Copyright 2015 CNI authors
+// Copyright 2017 rmohr@redhat.com
+// Copyright 2018 KubeVirt authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// GO version 1.10 or greater is required. Before that, switching namespaces in
+// long running processes in go did not work in a reliable way.
+
+// +build go1.10
```

Comment:
@davidvossel that makes sure that the code does not compile for versions older than 1.10.

Reply:
oh nice! I'm pretty good about this now.